networkit/cpp/graph/test/GraphGTest.cpp

```diff
@@ -132,6 +132,12 @@ void GraphGTest::SetUp() {
 
 /** CONSTRUCTORS **/
 
+TEST(GraphGTest, testDefaultConstructorIndexed) {
+    Graph G(3, false, false, true);
```

Comment:
Maybe test it for directed graphs as well?

Reply:
Good point.